release_notes.py

```diff
@@ -43,6 +45,7 @@ def loadData(self, data):
 
     def generateRN(self):
         res = ""
+        error_count = 0
```

Comment:
Very not critical - but if you only want to know if there was an error - you can use boolean value instead

(And give it more meaningful name - such has `missingReleseNotes`